apps/extension/src/core/domains/accounts/types.ts

```diff
@@ -62,13 +62,20 @@ type AccountJsonDcentOwnProperties = {
 
 export type AccountJsonDcent = AccountJson & AccountJsonDcentOwnProperties
 
+type AccountJsonSignetOwnProperties = {
+  signetUrl: string
```

Comment:
Since Signet can be self hosted, we want to make sure it's easy to identify which url a vault was imported from so users with many vaults can easily locate their vaults